pages/index.tsx

```diff
@@ -1,11 +1,61 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
 import { HomeLayout } from "../components/layout";
 import { SubmissionsTable } from "../components/submissions";
+import { Pagination, Submission } from "../interfaces";
 
 export default function HomePage() {
+  const [submissions, setSubmissions] = useState<Submission[]>([]);
+  const [pagination, setPagination] = useState<Pagination>({
+    count: 0,
+    total: 0,
+    perPage: 0,
+    currentPage: 0,
+    totalPages: 0,
+    links: {
+      next: "",
+      previous: "",
+    },
+  });
+  const [currentPage, setCurrentPage] = useState(1);
+  const [status, setStatus] = useState("");
+
+  useEffect(() => {
+    axios
+      .get(
```

Comment:
please use react-query instead of calling axios directly. We can have a chat on how to do this if you want to